features/sctp/Makefile

```diff
@@ -1,7 +1,9 @@
 .PHONY: test, ginkgo
 
+TESTERIMAGE?=fedepaol/sctptest:v1.0
```

Comment:
we can build this container in prow, and then execute it within the cluster.

see this for an example.
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml#L18

in ocs, we build some containers, like the ocs-registry container, and then we can deploy that container in our e2e tests. We'll know how to reference the container because the e2e test will set this ```OPENSHIFT_BUILD_NAMESPACE``` environment variable that can be used to retrieve the built image.

So, if your image was called ```sctptest```, you could build the image url with ```registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:sctptest```

Reply:
so, to further clarify, this approached involves

* creating a Dockerfile to build the sctptest container and putting that Dockerfile somewhere in the baremetal-deploy repo.
* have prow build that dockerfile in the `images` section of the corresponding ci-config.
* When the `make e2etest` target is called, we'll know how to reference the sctptest image using the OPENSHIFT_BUILD_NAMESPACE env var that will magically be put there for us in the prow job.

Reply:
Yes, that was my idea as well.
The Dockerfile is already in place (just at the bottom of the huuuuge file list), but I had this chicken-egg problem where I wanted to modify the job only when this was merged. That's the exactly the reason why I can pass the TESTERIMAGE value from outside. 

My plan was to override it in the prow job once this was ready.

As per the image, I thought about having this as a "base image" more than an "image". The difference here should be that images are expected to be exported for other jobs unless opting out. Base images are thrown away for the purpouse of the tests (which could also be building other images).